src/py42/services/users.py

```diff
@@ -242,10 +251,10 @@ def deactivate(self, user_id, block_user=None):
         Returns:
             :class:`py42.response.Py42Response`
         """
-        uri = f"/api/v1/UserDeactivation/{user_id}"
+        uri = f"/api/v3/users/{self._get_user_uid_by_id(user_id)}/deactivate"
         data = {"blockUser": block_user}
```

Comment:
the query params for this and reactivate got changed on the new apis to just `block` and `unblock` instead of `blockUser`/`unblockUser`

Reply:
Oh cool, they must be backwards compatible or something because they were working as is, but I'll update it to the new values 👍

Reply:
It wouldn't error, but the incorrect params would just be ignored. If you tried `deactivate(user_id, block_user=False)` (opposite of default), the user would still be blocked on deactivation.

Reply:
Ohh I see, I must've been testing it against the default values without knowing so the output looked right 🧐